docs/media/schematics/schematics.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="Electron" modified="2020-03-19T14:52:55.940Z" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/12.6.5 Chrome/80.0.3987.86 Electron/8.0.0 Safari/537.36" etag="owcDxc94sSXD17_qf3yT" version="12.6.5" type="device"><diagram id="uMucZhOneSB0JJpgZDKq" name="Running integration tests with Arcus">5Vpbc5s4FP41fnRGXITtx9RO28xkO9mms033TQYB2mLECuFLf/1KIC4CfEli7Ow0yQM6kg7S+b7v6EJG1ny1/cRQEv5BPRyNTOBtR9ZiZJqGbZoj+Qe8XWGZWsoQMOKpRrXhifzCygiUNSMeTrWGnNKIk0Q3ujSOscs1G2KMbvRmPo30tyYowB3Dk4uirvU78XioZgFBbf+MSRCWbzaAqlmhsrEypCHy6KZhsu5G1pxRyoun1XaOIxm8Mi5Fv497aquBMRzzUzr86z9kcPzl8ev93z+/bP56/mjf/jlWXtYoytSE1WD5rowAo1nsYekEjKwPm5Bw/JQgV9ZuBObCFvJVJEqGeFTuMON4u3ecRjV7QRtMV5iznWiiOtgqXoowRhm/TR3+qk3YDP1MGZGCPKhc11ERDyowLwiS2QnSE0eMC9NnmvIDATOOB8ynMVesN2xZJlE0pxFluS/Lh/JX2FPO6E/cqHHyH+WhYS9+zgOF04LC6oEC9EABh0Jidpyu2BP6VUXKeEgDGqPorrZ+0Aldt3mgNFGo/IM53ylYUMbpYcxENNnuWfnLCz9k4QaWxcW2WbnYqVIxdjngw/iI+dGMufhAYCyVGhELMD/QzunHm+EIcbLWx3F29KwOevcxxwETr6axqPiGh1aT75uu26cmz1k60BlGNbNrq8buyV+C6r9b+qoCfDUg4GsEIObP9aDrwYxpjFuRVyYUkSAWxQj70oOMJRE7m1tlXhHPyxNiH8Z6kuzALMtqiMYZkGqv+eDENd8cCimng9TdWs7FBJ/kvtV0IhnRJRNPAc/ZWlg8sm6b0gTFpe0bTYhbVoiBNesaZs1Liw5iG5nIx5RjiVOCGRFTxkyZHuvyUe2SLS533IYGeZ+ySzK5Igy5905uxc6e3DqZLQE4j6QtB+pMmcAOU6odY5Mp06GYMukwRYHcljEjKA726q2By2k4KFEzyovsUQg1pIz8Ej1Q6QwxV/WGoCCjS+LgIe+8MEAPjlMX9+O4nEIbnglHOANHcZxeEsbyvPnbbS2Pbhknb9wyqq6PlOT5s8z409bibEDdRbHlVb2ax9qWI2umZwTBLd1RMcGOo5wl1XzeQJzuEfoxy7upFaNFIjdj60rcOPZu5SWFzKsRSlPiHqHDlvCKDeK5IMMEqmJNBlnYNZnxXL2x4tDkgueTyYlksw5mjTG4MYGlwT22Z53EAQ/s6d5KW521Vjsfncpaw2g5slqO9rBWsAXtGs0S2SA9MF6r9R6oXRKJh8Jjq3fpnvp+ioeRTfcwOGcYcSxsT9kydRlJil3xEPoxNP2cKJ9G1v2h6aVfPR5Kw2qgZ5SS+ba8XUvJgLata8l0LqklB7SWAPuVS4A5aTmyJjenLQIvlRM09Tc570VN3SP+Akf4f6Qm40pqss+lJvlpQ+OGbVxSTG0NGM4rxeS0ljgTDCOl9gbQBu9FS91bmlvmZmIq4IEsGVKHjVd/Innxwfqqd2iG3Q/T0QO3PdhJrXvi/opRNP5OWdRzM3Mf+wyJ0GUuzxi+MHQXuRNpS/8dQDTtQCSvNkkcCONVAbnI5YYN3h0g3Q9nRz62DKKGS3x9gdblgi+K9ff7Yv2p/wvCuvsP</diagram></mxfile>
```

Comment:
Not sure what this is?

Reply:
This is the schematic file used to generate the image